Automatic Dependency Submission fails: wrong class version

## Build failure

<img width="527" height="203" alt="Image" src="https://github.com/user-attachments/assets/80e1907e-bc5a-4fbd-9d7d-851b8d125125" />

> FAILURE: Build failed with an exception.
> 
> * Where:
> Initialization script '/home/runner/.gradle/init.d/gradle-actions.build-result-capture.init.gradle'
> 
> * What went wrong:
> Could not compile initialization script '/home/runner/.gradle/init.d/gradle-actions.build-result-capture.init.gradle'.
> > startup failed:
>   General error during conversion: Unsupported class file major version 65

## Reason
The Automatic Dependency Submission action fails as it's using Java 21, which creates version 65 class files. 

## Possible solution
Applying a specific Java toolchain, in `build.gradle`, may fix the issue.

--- a/build.gradle
+++ b/build.gradle
@@ -47,6 +47,13 @@ dependencies {
   codenarc('org.apache.groovy:groovy-all:4.0.6') // Required for Java 17
 }
 
+// Apply a specific Java toolchain to ease working on different environments.
+java {
+  toolchain {
+    languageVersion = JavaLanguageVersion.of(17)
+  }
+}
+
 application {
   mainClass = 'net.ebdon.yttozara.YtToZara'
 }

